src/components/directory-content.js

```diff
@@ -98,6 +98,38 @@ const styles = {
         maxWidth: '250px',
         maxHeight: '50px',
     },
+    noContentContainer: (theme) => ({
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginTop: theme.spacing(10),
+    }),
+    noContentCircle: (theme) => ({
+        position: 'relative',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: 200,
+        height: 200,
+        backgroundColor: theme.row.primary,
+        borderRadius: theme.spacing(15),
+    }),
+    noContentIcon: {
+        fontSize: '100px',
```

Comment:
I think I would make a component which only contain this icon is a circle

Then I would create a constant "CIRCLE_SIZE"
Then here, width and height would be equal to "CIRCLE_SIZE"
"fontSize" would be equals to "CIRCLE_SIZE / 2"

I would make it easier to reuse and isolate complex CSS

Reply:
Done